### PR TITLE
Reformat docstring for `get_timestep_embedding`

### PR DESCRIPTION
**Pull request by alanhdu**
The original docstring formatting was off (looks like a bad line-wrap mangling the parameters), but I figured I might as well reformat to match the same docstring style as the rest of the file.

# What does this PR do?

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/diffusers/models/embeddings.py [hunk 1]
```diff
@@ -35,10 +35,15 @@ def get_timestep_embedding(
     """
     This matches the implementation in Denoising Diffusion Probabilistic Models: Create sinusoidal timestep embeddings.
 
-    :param timesteps: a 1-D Tensor of N indices, one per batch element.
-                      These may be fractional.
-    :param embedding_dim: the dimension of the output. :param max_period: controls the minimum frequency of the
-    embeddings. :return: an [N x dim] Tensor of positional embeddings.
+    Args
```
Comment (yiyixuxu) [inline src/diffusers/models/embeddings.py:38]: thanks!
can we also add the other 2 arguments that are missing from the doc string?

Comment (alanhdu) [reply src/diffusers/models/embeddings.py:38]: I can try -- TBH I haven't examined the function closely to really understand what `downscale_freq_shift` and `scale` actually do.

File: src/diffusers/models/embeddings.py [hunk 2]
```diff
@@ -35,25 +35,31 @@ def get_timestep_embedding(
...
+            The minimum frequency of the embeddings.
+        max_period (int):
+            Controls the maximum frequency of the embeddings
+    Returns
+        torch.Tensor: an [N x dim] Tensor of positional embeddings.
     """
     assert len(timesteps.shape) == 1, "Timesteps should be a 1d-array"
 
     half_dim = embedding_dim // 2
-    exponent = -math.log(max_period) * torch.arange(
```
Comment (yiyixuxu) [inline src/diffusers/models/embeddings.py:46]: can we not change the implementation?

Comment (alanhdu) [reply src/diffusers/models/embeddings.py:46]: Sure -- I find the current implementation to be needlessly complex, but hapy to revert.

File: src/diffusers/models/embeddings.py [hunk 3]
```diff
@@ -35,10 +35,21 @@ def get_timestep_embedding(
...
+        timesteps (torch.Tensor):
+            a 1-D Tensor of N indices, one per batch element. These may be fractional.
+        embedding_dim (int):
+            the dimension of the output.
+        flip_sin_to_cos (bool):
+            Whether the embedding order should be `cos, sin` (if True) or `sin, cos` (if False)
+        downscale_freq_shift (float):
+            Controls the delta between frequencies between dimensions
+        scale (float):
+            The minimum frequency of the embeddings.
```
Comment (yiyixuxu) [inline src/diffusers/models/embeddings.py:48]:
```suggestion
            Scaling factor applied to the embeddings.
```

**Comment by yiyixuxu** [comment]
thank you for your contribution!

**Pull request merged**